packages/office-ui-fabric-react/src/components/Button/MessageBarButton/MessageBarButton.tsx

```diff
@@ -3,15 +3,31 @@ import { DefaultButton } from '../DefaultButton/DefaultButton';
 import { BaseComponent, customizable, nullRender } from '../../../Utilities';
 import { IButtonProps } from '../Button.types';
 import { getStyles } from './MessageBarButton.styles';
+import { ITheme, getTheme } from '../../../Styling';
 
 /**
  * {@docCategory MessageBar}
  */
 @customizable('MessageBarButton', ['theme', 'styles'], true)
 export class MessageBarButton extends BaseComponent<IButtonProps, {}> {
+  private _globalTheme: ITheme;
+
+  constructor(props: IButtonProps) {
+    super(props);
+
+    this._globalTheme = getTheme();
+  }
+
   public render(): JSX.Element {
-    const { styles, theme } = this.props;
+    const { styles } = this.props;
 
-    return <DefaultButton {...this.props} styles={getStyles(theme!, styles)} onRenderDescription={nullRender} />;
+    return (
+      <DefaultButton
+        {...this.props}
+        styles={getStyles(this._globalTheme, styles)}
```

Comment:
This isn't always going to work. Theoretically we should have hardcoded colors for everything inside of the message bar, because if the background is hardcoded then all the foreground colors need to be hardcoded. However, I think this is still better than what it was before.